backend/app/llm/manager.py

```
@@ -0,0 +1,90 @@
+        
+        try:
+            primary_llm = self._create_llm(target_provider, **kwargs)
+            
+            # Build fallback chain for runtime errors
+            fallback_llms = self._build_fallbacks(target_provider, **kwargs)
+            
+            if fallback_llms:
+                return primary_llm.with_fallbacks(fallback_llms)
+            return primary_llm
+            
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)